drivers/sensor/bosch/bmp388/bmp388.c

```diff
@@ -490,7 +491,7 @@ static int bmp388_init(const struct device *dev)
 		return -EIO;
 	}
 
-	if (val != BMP388_CHIP_ID) {
+	if (val != bmp388->chip_id) {
```

Comment:
Maybe rename to `bmp3xx`?

Reply:
yes good idea to rename it just in this function only.